freeipa/src/main/resources/freeipa-salt/salt/freeipa/templates/freeipa_backup_hourly.j2

```diff
@@ -0,0 +1,6 @@
+{%- from 'freeipa/settings.sls' import freeipa with context -%}
+#!/bin/bash
+if [ ! -f /tmp/freeipa_backup_hourly_bypass ]; then
```

Comment:
The file does not exist by default.  It is looked for to allow support to disable the hourly and monthly backups temporarily while a restore is done.  The freeipa backup only supports a specific /var/lib/ipa/backup directory.  We want to remove old backups from the local.  If the backups are running while someone is trying to do a restore it will remove the local restore they are trying to do.